Replaced wrong 800x480 resolution .bmp files for AM070 machine with corrected 1280x800 resolution files

glidero opened the pull request.

kedder commented on README.md:
```diff
@@ -1,53 +0,0 @@
-# meta-openvario
```
Is this file removed by accident?

glidero commented:
Hi Andrey,well, yes and no. I made modification to readme.md two weeks ago (together with adding AM070 machine). I added "sudo" before docker, note about new machine AM070 and note about 2 possibilities of compilation - bitbake openvario-image or bitbake openvario-image. The last note was rejected, therefore I deleted the file so it doesn't seems I'm trying to force this modification again even it was already rejected. This pull request is just about replacing 8 .bmp files, which I uploaded in original 800x480 resolution 2 weeks ago by mistake.Sorry when I messed something up.DanielDňa Tue, 02 Feb 2021 02:09:49 -0800 Andrey Lebedev  napísal:> @kedder commented on this pull request.> > > >> @@ -1,53 +0,0 @@> -# meta-openvario> > Is this file removed by accident?
 > > -- > You are receiving this because you authored the thread.> Reply to this email directly or view it on GitHub:> https://github.com/Openvario/meta-openvario/pull/117#pullrequestreview-581177482

glidero commented:
Hi Andrey,I put the readme.md back to my fork. It's the same as it's now in main branch.DanielDňa Tue, 02 Feb 2021 02:09:49 -0800 Andrey Lebedev  napísal:> @kedder commented on this pull request.> > > >> @@ -1,53 +0,0 @@> -# meta-openvario> > Is this file removed by accident?> > -- > You are receiving this because you authored the thread.> Reply to this email directly or view it on GitHub:> https://github.com/Openvario/meta-openvario/pull/117#pullrequestreview-581177482

kedder reviewed (approved):
@glidero, thanks, it looks good now.

@linuxianer99 feel free to merge. I recommend using "Squash and Merge", because there are lots of fixup commits.
